### ISSUE TITLE
Problems publishing to Movable Type

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#105]
```
What's the problem?
Only the title posts - not the content of the post
neither draft nor schedule mode works. Publishes no matter what
I am using Movable Type 4.3

What browser are you using? Firefox 3.6.8

What version of ScribeFire are you running? Scribefire Next

On the Feature side, it would be nice to be able to select categories.

Also running Windows XP on this PC

rowan@uncommonthought.com

```

-----
Original issue reported on code.google.com by `rowanw...@gmail.com` on 24 Jul 2010 at 9:29

**Comment by GoogleCodeExporter** [comment]
```
I believe that these problems were fixed at the same time that support for 
Drupal was fixed; I'll revisit after version 1.2.
```

-----
Original comment by `cfinke` on 25 Jul 2010 at 7:42
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
Install version 1.2.0.1 of ScribeFire Next; I believe it should fix problems 
with Movable Type.

https://addons.mozilla.org/en-US/firefox/addon/199790/
```

-----
Original comment by `cfinke` on 29 Jul 2010 at 3:16
* Changed state: **Closed**
